scripts/covid19_india/medical_tests_in_data/COVID19_tests_india.tmcf

```diff
@@ -0,0 +1,7 @@
+Node: E:COVID19_tests_india->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19
+measurementMethod: dcs:CovidTrackingProject
```

Comment:
Should the measurementMethod be something else? CovidTrackingProject refers to https://covidtracking.com/

Reply:
I can make it `Covid19IndiaProject`. 

I assumed CovidTrackingProject was a generic name for all  covid19 related projects.

Reply:
Will setup yapf.